pkg/virtualKubelet/liqoNodeProvider/nodeProvider.go

```diff
@@ -15,6 +15,9 @@ type LiqoNodeProvider struct {
 	client    kubernetes.Interface
 	dynClient dynamic.Interface
 
+	node      *corev1.Node
+	oldLabels map[string]string
```

Comment:
I changed the name, but this can not be removed, since it stores the last labels provided by the advertisement/resourceOffer. We need to track them to prevent the deletion of the labels that the admin of the cluster may add to our virtual node